2.evaluate_data/README.md

```diff
@@ -0,0 +1,18 @@
+# Apply phenotypic profiling model to JUMP data
+
+In this module, we generate single-cell probabilities for each of the 15 phenotypic classes by applying the [phenotypic profiling model](https://github.com/WayScience/phenotypic_profiling_model).
+There are two model types, final and shuffled baseline, where single-cell features are randomized independently. 
```

Comment:
```suggestion
There are two model types, final and shuffled baseline.
The shuffled baseline model trains using randomly shuffled single-cell features. 
```

To increase clarity and move to active voice.

Reply:
👍